src/com/fediroryshchuk/homework3/carconstructor/CarDoor.java

```diff
@@ -0,0 +1,60 @@
+package com.fediroryshchuk.homework3.carconstructor;
+
+public class CarDoor {
+    private boolean doorClose;
+    private boolean windowClose;
+
+    public CarDoor() {
+        doorClose = true;
+        windowClose = true;
+    }
+
+    public CarDoor(boolean door, boolean window) {
+        this.windowClose = window;
+        this.doorClose = door;
+    }
+
+    public void openDoor() {
+        if (doorClose) {
```

Comment:
If doorClose is false it will not change and it will be false. If it true it will change and it will be false.
Simplification:  this.doorClose=false;